lib/beta.py

```diff
@@ -0,0 +1,34 @@
+import sublime
+from ..lib import settings, app_controller, requests 
+from ..lib.link_opener import open_copilot_root
+from ..lib.languages import Languages, Extensions, ext_to_lang, kited_ext_enabled
+
+_BETA_EXTS = (Extensions.GO, Extensions.JS, Extensions.JSX, Extensions.VUE)
+
+_LANG_TO_SHOW_NOTIF_KEY = {
+  Languages.GO: "show_go_beta_notif", 
```

Comment:
Couple of stylistic nits here:

* The code base uses 4 spaces instead of 2 for tabs
* The code base prefers single `'` to double `"`